mysql/errcode.go

```diff
@@ -934,48 +934,61 @@ const (
-	ErrInvalidPluginSysVarName    = 8106
-	ErrRequireVersionCheckFail    = 8107
-	ErrUnsupportedReloadPlugin    = 8018
-	ErrUnsupportedReloadPluginVar = 8019
-	ErrTableLocked                = 8020
-	ErrNotExist                   = 8021
-	ErrTxnRetryable               = 8022
-	ErrCannotSetNilValue          = 8023
-	ErrInvalidTxn                 = 8024
-	ErrEntryTooLarge              = 8025
-	ErrNotImplemented             = 8026
-	ErrInfoSchemaExpired          = 8027
-	ErrInfoSchemaChanged          = 8028
-	ErrBadNumber                  = 8029
-	ErrCastAsSignedOverflow       = 8030
-	ErrCastNegIntAsUnsigned       = 8031
-	ErrInvalidYearFormat          = 8032
-	ErrInvalidYear                = 8033
-	ErrIncorrectDatetimeValue     = 8034
-	ErrInvalidTimeFormat          = 8036
-	ErrInvalidWeekModeFormat      = 8037
-	ErrFieldGetDefaultFailed      = 8038
-	ErrIndexOutBound              = 8039
-	ErrUnsupportedOp              = 8040
-	ErrRowNotFound                = 8041
-	ErrTableStateCantNone         = 8042
-	ErrColumnStateNonPublic       = 8043
-	ErrIndexStateCantNone         = 8044
-	ErrInvalidRecordKey           = 8045
-	ErrColumnStateCantNone        = 8046
-	ErrUnsupportedValueForVar     = 8047
-	ErrUnsupportedIsolationLevel  = 8048
+	ErrMemExceedThreshold          = 8001
+	ErrForUpdateCantRetry          = 8002
+	ErrAdminCheckTable             = 8003
+	ErrTxnTooLarge                 = 8004
+	ErrWriteConflictInTiDB         = 8005
+	ErrInvalidPluginID             = 8101
+	ErrInvalidPluginManifest       = 8102
+	ErrInvalidPluginName           = 8103
+	ErrInvalidPluginVersion        = 8104
+	ErrDuplicatePlugin             = 8105
+	ErrInvalidPluginSysVarName     = 8106
+	ErrRequireVersionCheckFail     = 8107
+	ErrUnsupportedReloadPlugin     = 8018
+	ErrUnsupportedReloadPluginVar  = 8019
+	ErrTableLocked                 = 8020
+	ErrNotExist                    = 8021
+	ErrTxnRetryable                = 8022
+	ErrCannotSetNilValue           = 8023
+	ErrInvalidTxn                  = 8024
+	ErrEntryTooLarge               = 8025
+	ErrNotImplemented              = 8026
+	ErrInfoSchemaExpired           = 8027
+	ErrInfoSchemaChanged           = 8028
+	ErrBadNumber                   = 8029
+	ErrCastAsSignedOverflow        = 8030
+	ErrCastNegIntAsUnsigned        = 8031
+	ErrInvalidYearFormat           = 8032
+	ErrInvalidYear                 = 8033
+	ErrIncorrectDatetimeValue      = 8034
+	ErrInvalidTimeFormat           = 8036
+	ErrInvalidWeekModeFormat       = 8037
+	ErrFieldGetDefaultFailed       = 8038
+	ErrIndexOutBound               = 8039
+	ErrUnsupportedOp               = 8040
+	ErrRowNotFound                 = 8041
+	ErrTableStateCantNone          = 8042
+	ErrColumnStateNonPublic        = 8043
+	ErrIndexStateCantNone          = 8044
+	ErrInvalidRecordKey            = 8045
+	ErrColumnStateCantNone         = 8046
+	ErrUnsupportedValueForVar      = 8047
+	ErrUnsupportedIsolationLevel   = 8048
+	ErrUnsupportedType             = 8049
```

Comment:
It is better to start from 8100 to avoid some conflicts.

Reply:
OK, will do

Reply:
@jackysp but it seems starting from 8100 has already been taken

```
	ErrInvalidPluginID             = 8101
	ErrInvalidPluginManifest       = 8102
	ErrInvalidPluginName           = 8103
	ErrInvalidPluginVersion        = 8104
	ErrDuplicatePlugin             = 8105
	ErrInvalidPluginSysVarName     = 8106
	ErrRequireVersionCheckFail     = 8107
```

So do we want to start from 8108 or 8200?